sdk/openai/azure-ai-openai/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 ### Breaking Changes
+- Removed `NonAzureOpenAIKeyCredential` and `NonAzureOpenAIKeyCredentialPolicy`. Replace by Azure Core `KeyCredential` and `KeyCredentialPolicy`.
```

Comment:
`NonAzureOpenAIKeyCredentialPolicy` was in implementation package. So, that should not be mentioned in changelog.